package.json

```diff
@@ -143,7 +143,6 @@
     "audit-ci": "^7.0.1",
     "cheerio": "^1.0.0-rc.12",
     "concurrently": "^8.2.2",
-    "cookie-session": "^2.1.0",
```

Comment:
does this want to be taken off?

Reply:
Yeah, it wasn't used apart from in test setup and not even there really. It's gone from the template app.